app/controllers/password_resets_controller.rb

```diff
@@ -57,9 +57,11 @@ def valid_user
 
     # Checks expiration of reset token.
     def check_expiration
-      if @user.password_reset_expired?
-        flash[:danger] = "Password reset has expired."
-        redirect_to new_password_reset_url
+      if @user.reset_sent_at
```

Comment:
> __[RuboCop]__ Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression. ([link](https://sider.review/gh/repos/152823695/pulls/5#issue-3531353))

please fix